packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -278,18 +290,29 @@ class ExpoImageView(
 
       expoImageViewWrapper.get()?.onLoadStart?.invoke(Unit)
 
+      val newTarget = target.getUnusedTarget() ?: return
+
       val placeholder = bestPlaceholder?.createGlideModel(context)
       val request = requestManager
         .asDrawable()
         .load(sourceToLoad.glideData)
-        .apply { if (placeholder != null) thumbnail(requestManager.load(placeholder.glideData)) }
+        .apply {
+          if (placeholder != null) {
+            thumbnail(requestManager.load(placeholder.glideData))
+            val placeholderContentFit = if (bestPlaceholder?.isBlurhash() == true) {
+              ContentFit.Cover
```

Comment:
Actually for blurhashes iOS uses the same content fit as the proper image because `cover`, `contain` and `fill` would work well so it gives more flexibility until we add `placeholderContentFit`